src/roles/vsd-deploy/tasks/main.yml

```diff
@@ -375,6 +375,13 @@
         /opt/vsd/vsd-install.sh -1 {{ first_cluster_node_to_install }} -t 3
         -x {{ vsd_fqdn }} -y {{ (enable_ipv6 | default(False)) | ternary('-6', '') }}
       when: inventory_hostname == third_cluster_node_to_install
+    
```

Comment:
We have an existing role that blocks xmpp connections to the VSD. Can that be used here? It's called "block-xmpp-connections"

Reply:
done